[ET][Portable][Build Size] Reduce build size of op_pixel_shuffle & op_pixel_unshuffle

manuelcandales opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #6021
* __->__ #6020
* #6019
* #6018
* #6017
* #6016
* #6015
* #6014
* #6013
* #6012
* #6011
* #6010
* #6009
* #6008
* #6007
* #6006
* #6005

pixel_shuffle:     60 K -> 1.8 K
pixel_unshuffle: 57 K -> 1.7 K

Differential Revision: [D63994872](https://our.internmc.facebook.com/intern/diff/D63994872/)

manuelcandales committed:
[ET][Portable][Build Size] Reduce build size of op_pixel_shuffle & op_pixel_unshuffle

pixel_shuffle:     60 K -> 1.8 K
pixel_unshuffle: 57 K -> 1.7 K

Differential Revision: [D63994872](https://our.internmc.facebook.com/intern/diff/D63994872/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_pixel_shuffle & op_pixel_unshuffle"


pixel_shuffle:     60 K -> 1.8 K
pixel_unshuffle: 57 K -> 1.7 K

Differential Revision: [D63994872](https://our.internmc.facebook.com/intern/diff/D63994872/)

[ghstack-poisoned]

malfet reviewed (approved):
Nice! (though unsure why it needs to be part of the stack)

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_pixel_shuffle & op_pixel_unshuffle"


pixel_shuffle:     60 K -> 1.8 K
pixel_unshuffle: 57 K -> 1.7 K

Differential Revision: [D63994872](https://our.internmc.facebook.com/intern/diff/D63994872/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_pixel_shuffle & op_pixel_unshuffle"


pixel_shuffle:     60 K -> 1.8 K
pixel_unshuffle: 57 K -> 1.7 K

Differential Revision: [D63994872](https://our.internmc.facebook.com/intern/diff/D63994872/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_pixel_shuffle & op_pixel_unshuffle"


pixel_shuffle:     60 K -> 1.8 K
pixel_unshuffle: 57 K -> 1.7 K

Differential Revision: [D63994872](https://our.internmc.facebook.com/intern/diff/D63994872/)

[ghstack-poisoned]
